src/mlpack/tests/data/impute_test.csv

```diff
@@ -0,0 +1,3 @@
+a, 2, 3
```

Comment:
The file is created internally inside the unit test.
I forgot to erase this file after testing it manually. sorry about that.